### ISSUE TITLE
Voicemail App Multi Language

### ISSUE THREAD

**Issue by volga629-1** [volga629/fusionpbx-dev#28]
Voice Mail Application contain bug where multi language is mix up.  

Recording

https://www.dropbox.com/s/xz3bnlb1k56ioub/7-29-2019%205.37.19PM.mp4?dl=0

**Comment by volga629-1** [comment]
Default Settings introduce in Domain section 3 new variables 

1. domain_language 
2. domain_voice
3. domain_dialect 


Test case 
If  variables are NOT present in domain level we fall back to Default Settings variables

**Comment by ankitvasava98** [comment]
Hello Slava,

I have finished that 
below part
Default Settings introduce in Domain section 3 new variables

domain_language
domain_voice
domain_dialect
Test case
I have question 

if above 3 variable didn't set then we can get that 3 variable value from default setting

**means we can also set that 3 variable into default settings? right?**

**Comment by volga629-1** [comment]
Yes default values will be same freeswitch var en callie us, please check make sure it set correctly.

**Comment by volga629-1** [comment]
Feature completed, language domain variables works.
Initial language folder setup will be done through install scripts. 
I will do initial commit of install scripts.
I will keep ticket open for future testing.

 Last commit

https://github.com/VoIP-SAAS/fusionpbx-dev/commit/c442a2ca13a6ebdf55a254c01aa3eff23c04905c
